src/message.c

```diff
@@ -367,6 +367,17 @@ static void handle_domain_config(FILE *rsp, struct token domain, char *message)
     } else {
       daemon_fail(rsp, "value for '%.*s' must be either 'main' or 'all'.\n", command.length, command.text);
     }
+  } else if (token_equals(command, COMMAND_CONFIG_BAR_POSITION)) {
+    int length = strlen(message);
+    char * top = "top";
+    char * bottom = "bottom";
```

Comment:
I'm [not personally a fan of / don't understand any advantages to] this pre-decleration of once-used string literals, but I just tried to follow your style as it is

Reply:
Yeah me neither, its fixed on master.